pkg/model/tests/data/bootstrapscript_2.txt

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+NODEUP_URL=NUSource
+NODEUP_HASH=NUSHash
+
+
+
+
+export http_proxy=http://example.com:80
+export https_proxy=${http_proxy}
+export no_proxy=
+export NO_PROXY=${no_proxy}
+echo "export http_proxy=${http_proxy}" >> /etc/default/docker
```

Comment:
Just a crazy side not we have functionality for this now in kops.  i.e. egressProxy will setup http proxy.